src/components/popover/bl-popover.stories.mdx

```diff
@@ -37,6 +37,11 @@ export const popoverOpener = async ({canvasElement}) => {
   }
 }
 
+export const nestedPopoverOpener = async ({canvasElement}) => {
+  await popoverOpener({ canvasElement });
+  await popoverOpener({ canvasElement: canvasElement.querySelector('#popover-nested-parent') });
```

Comment:
I think we can add a small delay before opening this second popover since it causes positioning failure on chromatic firefox screenshot. Maybe 100ms... Something like:

```ts
await new Promise((resolve) => setTimeout(resolve, 100));
```